Makefile

```diff
@@ -50,21 +58,25 @@ clean:
 	-$(RM) testdata/*.elf
 	-$(RM) internal/btf/testdata/*.elf
 
-all: $(addsuffix -el.elf,$(TARGETS)) $(addsuffix -eb.elf,$(TARGETS))
+all: $(addsuffix -el.elf,$(TARGETS)) $(addsuffix -eb.elf,$(TARGETS)) generate
 	ln -srf testdata/loader-$(CLANG)-el.elf testdata/loader-el.elf
 	ln -srf testdata/loader-$(CLANG)-eb.elf testdata/loader-eb.elf
 
+generate:
+	go generate ./cmd/bpf2go
```

Comment:
Could we put the `export` statements that were added above here instead? That would make it a bit more explicit and easier to track down where they're used.

Reply:
We can't make them part of the recipe, if that's what you mean (since the recipe runs in a shell already).

The best we could do is:

```
export FOO

generate:
```

Do you prefer that?